docs/reference/body-segmentation.md

```diff
@@ -158,9 +158,13 @@ const bodySegmentation = ml5.bodySegmentation(?modelName, ?options, ?callback);
 
 **Parameters:**
 
-- **modelName**: OPTIONAL: A string specifying which model to use, "SelfieSegmentation" or "BodyPix".
+- **modelName**: Optional. A string specifying which model to use. Types of model:
+  - _BodyPix_ (default): A general-purpose model that can be used to segment people and body parts.
```

Comment:
SelfieSegmentation is the default. Check https://github.com/ml5js/ml5-next-gen/blob/main/src/BodySegmentation/index.js#L31 for reference.